.github/workflows/sycl-windows-build.yml

```diff
@@ -50,6 +57,12 @@ on:
         type: choice
         options:
           - 3
+      compiler:
+        type: choice
+        options:
+          - cl
+          - icx
```

Comment:
Will it work without `build_configure_extra_args` for manual dispatch? Or am I missing something?

Reply:
we need this to pass the compiler flags we set in `sycl-post-commit.yml`, and then we use it an an envvar named `ARGS` in this file that is passed to `configure.py`, we need the compiler flags to get the thing to build

Reply:
talked offline, fixed in latest commit, thx